frontend/src/containers/RegisterUser.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom'
+import Button from 'react-bootstrap/Button'
+import "./Login.css";
+import Form from "react-bootstrap/Form";
+
+class RegisterUser extends Component {
+  constructor(props){
+    super(props);
+    this.state={
+      first_name:'',
+      last_name:'',
+      email:'',
+      password:''
+    }
+  }
+
+  setRedirect = () => {
+        this.setState({
```

Comment:
Small comment for future posterity.

Suggest adding a comment within method setRedirect() just prior to this.setState...

/* Add async exchange to determine login availability and eventually success/failure. Suggest using Auth0... example: https://auth0.com/blog/handling-authentication-in-react-with-context-and-hooks/ */

In the future we will not want method renderRedirect() to redirect unless the server has provided an auth token...